site/docs/contribute.md

```diff
@@ -45,6 +45,28 @@ The Iceberg community prefers to receive contributions as [Github pull requests]
 * If a PR is related to an issue, adding `Closes #1234` in the PR description will automatically close the issue and helps keep the project clean
 * If a PR is posted for visibility and isn't necessarily ready for review or merging, be sure to convert the PR to a draft
 
+### Merging Pull Requests
+
+Most pull requests can be merged once a committer is satisfied with the code in the PR. Before 
+merging all review comments should be addressed either by making changes or agreeing the request is out of scope for the PR. For larger changes or additions to public
+APIs committers will wait at least 24 hours before merging to ensure there is no additional feedback from members of the community. As in
+all ASF governed projects committers are expected to act in the best interest of the project. If a committer feels there might be a conflict
+of interest with a pull request they review, they are encouraged to ask for another committer to look at the pull request.
+
+A -1 vote on a PR (left as a comment with a 
+reason attached) indicates
```

Comment:
nit: could we remove the unnecessary newlines in this paragraph?

Reply:
done.